src/drupalmediatoolbar.js

```diff
@@ -0,0 +1,27 @@
+import { Plugin } from 'ckeditor5/src/core';
+import { WidgetToolbarRepository } from 'ckeditor5/src/widget';
+
+import { getSelectedDrupalMediaWidget } from './utils';
+
+export default class DrupalMediaToolbar extends Plugin {
+  static get requires() {
+    return [WidgetToolbarRepository];
+  }
+
+  static get pluginName() {
+    return 'DrupalMediaToolbar';
+  }
+
+  afterInit() {
+    const editor = this.editor;
+    const { t } = editor;
+    const widgetToolbarRepository = editor.plugins.get(WidgetToolbarRepository);
+
+    widgetToolbarRepository.register('drupalMedia', {
+      ariaLabel: t('Drupal Media toolbar'),
+      items: editor.config.get('drupalMedia.toolbar') || [],
```

Comment:
We should document how this API works. I don't know how to test this based on the current documentation.

Reply:
This allows us to add a `toolbar` property to the drupalMedia config, and it grabs any items in that array and registers them to this element. https://ckeditor.com/docs/ckeditor5/latest/api/module_widget_widgettoolbarrepository-WidgetToolbarRepository.html